charts/seed-bootstrap/charts/nginx-ingress/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: nginx-ingress
+    component: "controller"
+  name: nginx-ingress-controller
+  namespace: garden
+spec:
+  replicas: 3
+  revisionHistoryLimit: 0
+  selector:
+    matchLabels:
+      app: nginx-ingress
+      component: "controller"
+      release: addons
+  template:
+    metadata:
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+      labels:
+        app: nginx-ingress
+        component: "controller"
+        release: addons
+    spec:
+      affinity:
+        podAntiAffinity:
+          preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                topologyKey: kubernetes.io/hostname
+                labelSelector:
+                  matchExpressions:
+                    - key: app
+                      operator: In
+                      values:
+                        - nginx-ingress
+                    - key: component
+                      operator: In
+                      values:
+                        - controller
+                    - key: release
+                      operator: In
+                      values:
+                        - addons
+      priorityClassName: system-cluster-critical
```

Comment:
Hm, really? Should we remove that? The only consequence would be that logging/monitoring is not available, but "cluster critical" is definitely wrong. You can assign a dedicated priority > 0 instead.

Reply:
We could assign `gardener-shoot-controlplane` as the priorityClass, WDYT?
`// PriorityClassNameShootControlPlane is the name of a priority class for critical pods of a shoot control plane.`